frontend/scss/organisms/_o-custom-tours-viewer.scss

```diff
@@ -0,0 +1,126 @@
+// ------ Custom tours viewer
+
+// ---- Custom tours: Landing Page
+
+.aic-ct-landingpage__body {
+  padding-top: 32px;
+  padding-bottom: 32px;
+  background-color: $color__black--5;
+
+  &::before {
+    content: "";
+    position: absolute;
+    z-index: -1;
+    left: 50%;
+    top: 0;
+    bottom: 0;
+    width: 100vw;
+    margin-left: -50vw;
+    background-color: inherit;
+    pointer-events: none;
+  }
+
+  .title {
+    margin-bottom: 20px;
+  }
+
+  .aic-ct-grid-intro {
+    margin-bottom: 32px;
+    @include font-styles($f-body);
+  }
+}
+
+.aic-ct-grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 20px;
+
+  @include breakpoint('medium') {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @include breakpoint('small') {
+    grid-template-columns: 100%;
+  }
+
+  @include breakpoint('xsmall') {
+    grid-template-columns: 100%;
+  }
+
+  .aic-ct-grid-card {
+    height: fit-content;
+    background-color: $color__white;
+
+    .aic-ct-grid-card__top {
+      position: relative;
+      overflow: hidden;
+    }
+
+    .aic-ct-grid-card__overlay {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      background: linear-gradient(0deg, rgba(0, 0, 0, 0.42) 45%, rgba(0, 0, 0, 0.00) 100%);
+    }
+
+    .aic-ct-grid-card__title {
+      position: absolute;
+      left: 20px;
+      bottom: 20px;
+      margin-top: 20px;
+      margin-right: 20px;
+      color: $color__white;
+      font-family: $serif-font--loaded;
+      font-weight: 300;
+      font-size: 28px;
+      line-height: 1.14;
+      letter-spacing: -0.01em;
+    }
+
+    .aic-ct-grid-card__img {
+      aspect-ratio: 67/40;
+      height: auto;
+      width: 100%;
+    }
+
+    .default-img {
+      padding: 0 !important;
```

Comment:
Check: Can the padding be removed?

Reply:
I've double checked and actually it can't be removed.